sdk/src/mmmClient.ts

```diff
@@ -58,6 +64,26 @@ export const MMMProgramID = new PublicKey(
   'mmm3XBJg5gk8XJxEKBvdgptZz6SgK4tXvn36sodowMc',
 );
 
+export function isMplCoreAsset(
+  account: anchor.web3.AccountInfo<Buffer>,
+): boolean {
+  return account?.owner.equals(new PublicKey(MPL_CORE_PROGRAM_ID)) ?? false;
```

Comment:
MPL_CORE_PROGRAM_ID is already a PublicKey, right? so we don't need a `new` here

Reply:
it is a separate PublicKey type defined in mpl core package, need to convert to web3 type here